processing/src/main/java/io/druid/segment/serde/ComplexColumnPartSerde.java

```diff
@@ -59,21 +58,110 @@ public String getTypeName()
     return typeName;
   }
 
-  @Override
-  public long numBytes()
+  public static SerializerBuilder serializerBuilder()
```

Comment:
It seems a little strange to me is the "Serde" name. What i thought before this is a abbreviation of serializer and deserializer, does it?
So i think ColumnPartSerde is a combination of Serializer and Deserializer, when we create object, it's either a Serializer or Deserializer